styles.css

```diff
@@ -0,0 +1,462 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,700;1,800&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+body {
+  background-color: #f7f7f9;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 2.5rem;
+  background-color: #fff;
+  height: 50px;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  line-height: 20px;
+  text-decoration: none;
+  padding-left: 2rem;
+}
+
+.button {
+  margin-right: 2rem;
+}
+
+#main-section {
+  background-color: #fff;
+  border-bottom-left-radius: 50px;
+}
+
+.about {
+  height: 625px;
+  background-image: url("images/header-shapes mobile.png");
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.3rem;
+  padding-top: 5rem;
+}
+
+.about h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  padding: 1.5rem 0 0 1.5rem;
+  margin-top: 3.3rem;
+  margin-bottom: 0.7rem;
+}
+
+.about p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex-wrap: wrap;
+  padding: 0 3rem 0.3rem 1.5rem;
+  margin-bottom: 0.3rem;
+}
+
+li a {
+  margin: 0;
+  padding: 0 !important;
+}
+
+.about a {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-decoration: none;
+  padding: 0.5rem 1.5rem 0.4rem 1.5rem;
+}
+
+.socialmedia {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  flex: wrap;
+  margin: 0 4rem 13rem 1.4rem;
+  gap: 1rem;
+}
+
+/* WORK SECTION */
+
+.work-section {
+  display: flex;
+  flex-direction: column;
+  padding: 7.125rem 1.5rem;
+  gap: 5.5rem;
+}
```

Comment:
- [ ] As demanded by the project requirements, kindly use CSS grid to lay out the container of all your work cards. Grid and flexbox aid in the responsiveness of your application, it is important you learn how to use both.

Reply:
Noted with thanks @Kamba56. I will make the necessary changes 👌🏾